src/status/libstatus/accounts/constants.nim

```diff
@@ -170,36 +171,45 @@ var NODE_CONFIG* = %* {
 
 const DEFAULT_NETWORK_NAME* = "mainnet_rpc"
 
-let sep = if defined(windows): "\\" else: "/"
+const sep = when defined(windows): "\\" else: "/"
 
-let homeDir = getHomeDir()
-
-let parentDir =
-  if getEnv("NIM_STATUS_CLIENT_DEV").string != "":
-    "."
-  elif homeDir == "":
-    "."
-  elif defined(macosx):
-    joinPath(homeDir, "Library", "Application Support")
-  elif defined(windows):
-    let targetDir = getEnv("LOCALAPPDATA").string
-    if targetDir == "":
-      joinPath(homeDir, "AppData", "Local")
-    else:
-      targetDir
-  else:
-    let targetDir = getEnv("XDG_CONFIG_HOME").string
-    if targetDir == "":
-      joinPath(homeDir, ".config")
+proc defaultDataDir(): string =
+  let homeDir = getHomeDir()
+  let parentDir =
+    if defined(development):
+      parentDir(getAppDir())
+    elif homeDir == "":
```

Comment:
Is the home dir empty when in local development mode? I'm not sure it's clear when this case would exist.